css/global.css

```diff
@@ -840,6 +945,50 @@ Media Queries (overriding styles for different screen sizes)
 	}
 }
 
+/* Tablet and smaller 1024px*/
```

Comment:
@SolivanLau I had a merge conflict with two versions of these styles. Can you please verify that these are the correct styles? I saved the other block if you need to see it. Thanks!

Reply:
Yes. just to confirm, the **.section-2-col** styles are applied to 2 pages(values and impact)

They were moved to **global.css** from a **page.css**